rollup.config.js

```diff
@@ -3,31 +3,87 @@ import { terser } from "rollup-plugin-terser";
 
 export default [
   {
-    input: "src/single-spa-react.js",
+    input: `src/single-spa-react.js`,
+    output: [
+      {
+        file: `lib/umd/single-spa-react.js`,
+        format: "umd",
+        name: "singleSpaReact",
+        sourcemap: true,
+      },
+      {
+        file: `lib/system/single-spa-react.js`,
+        format: "system",
+        sourcemap: true,
+      },
+      {
+        file: `lib/esm/single-spa-react.js`,
+        format: "esm",
+        sourcemap: true,
+      },
+    ],
+    plugins: [babel(), terser()],
```

Comment:
Let's upgrade to rollup-plugin-terser@6 as part of this

Reply:
Ok